prow/tide/tide.go

```diff
@@ -334,6 +335,18 @@ func (sc *statusController) setStatuses(all, pool []PullRequest) {
 			}
 		}
 		if wantState != strings.ToLower(string(actualState)) || wantDesc != actualDesc {
+			var targetUrl string
+			if tideUrl := sc.ca.Config().Tide.TargetURL; tideUrl != "" {
+				targetUrl = tideUrl
+			} else if baseUrl := sc.ca.Config().Tide.PRStatusBaseUrl; baseUrl != "" {
+				parsedUrl, err := url.Parse(baseUrl)
+				if err != nil {
+					log.WithError(err).Error("Failed to parse PR status base URL")
+				} else {
+					parsedUrl.Query().Set("query", fmt.Sprintf("is:pr state:open author:%s head:%s", pr.Author.Login, pr.HeadRefName))
```

Comment:
We should drop `state:open` so that the link still works after the PR is merged.

You should also include `repo:` here in case someone makes a branch and creates PRs in multiple repos from it (unlikely, but possible).

Since we can't search by PR number I think using `author:` and `head:` is appropriate.

Reply:
Yeah, the tuple of `(repo, head, author)` must be unique -- GitHub won't let you make two PRs from one `head`.